lisp/lisprb.lisp

```diff
@@ -233,11 +221,11 @@
               (sphere-new #.(v 0.0 5.0 -1.0) 4.0 #.(v 1.0 0.0 0.0) nil)
               (sphere-new #.(v 8.0 5.0 -1.0) 2.0 #.(v 0.0 0.0 1.0) nil))))
 
-(defun world-camera (world)
```

Comment:
This is an anti-pattern. Use inlining instead

Reply:
Fine. Code updated.